src/sage/rings/integer.pyx

```diff
@@ -2951,13 +2951,15 @@ cdef class Integer(sage.structure.element.EuclideanDomainElement):
 
         return n
 
-    def prime_divisors(self):
+    def prime_divisors(self, *, limit=None):
```

Comment:
Why the `*`?

Reply:
I thought the syntax `n.prime_divisors(100)` that would otherwise be possible is not descriptive enough on its own. But I have no strong opinion about it, and indeed making arguments keyword-only is currently unusual in Sage.